tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1213,6 +1215,42 @@ static void VerifyPdbUtil(dxc::DxcDllSupport &dllSupport,
     TestArgumentPair(Args, ExpectedArgs);
   }
 
+  // Shader reflection
+  if (TestReflection) {
+    CComPtr<IDxcContainerReflection> pContainerRef;
+    VERIFY_SUCCEEDED(dllSupport.CreateInstance(CLSID_DxcContainerReflection, &pContainerRef));
+    VERIFY_SUCCEEDED(pContainerRef->Load(pBlob));
```

Comment:
We should make `IDxcUtils::CreateReflection` work with PDB as well as with the DxilContainer from the PDB if possible.

Reply:
Done.